### ISSUE TITLE
2022 Swin Transformer V2: Scaling Up Capacity and Resolution

### ISSUE THREAD

**Issue by pomelyu** [pomelyu/paper-reading-notes#19]
## Introduction
This paper introduces a modified swin transformer which is designed to scale up to 3B parameters and be capable of training on images up to 1536x1536.  It also proposes techniques to tackle three major issues in training and application of large vision models,  including training instability,  resolution gaps between pre-training and fine-tuning, and hunger on  labelled  data. Compared to other billion level model, swin transformer v2 requires much less data and training time.

## Method
- use residual-post-norm method combined with cosine attention to improve training stability
- use log-scale continuous position encoding to transfer the pre-trained model trained on low-resolution to fine-tuning model on high-resolution.
- use self-supervised method (SimMIM[^1]) to reduce the needs for vast labeled data.
- adopt several techniques to reduce the memory consumption, includes zero-optimizer[^2], activation check-pointing[^3] and their modified version of self-attention computation, with marginal effect to training speed.

## Comments

[^1]: Simmim:  A simple framework for masked image modeling
[^2]: Zero:  Memory optimizations toward training trillion parameter models
[^3]: Training deep nets with sublinear memory cost
